pkg/web/templates/login.html

```diff
@@ -30,7 +30,7 @@ <h1 class="pb-2 text-xl font-semibold">Log into your TRISA Node</h1>
         </div>
         <input type="hidden" id="next" name="next" value="" />
         <div class="py-2">
-          <button type="submit" id="login-bttn" class="btn text-base w-full bg-[#517994] text-white hover:bg-[#517994]/90">Log In</button>
+          <button type="submit" id="login-bttn" class="btn text-base w-full bg-primary text-white hover:bg-primary/90">Log In</button>
```

Comment:
Here's a link to where the primary color has been set in the Tailwind config file: https://github.com/trisacrypto/envoy/blob/main/tailwind.config.js#L16